File: 2024/src/d00.ts
import { readFileSync } from "fs";

export class Solver {
  constructor(private readonly input: string) {}

  part1() {
    return 0;
  }

  part2() {
    return 0;
  }

  private parse() {
    return this.input.trim().split("\n");
  }
}

if (require.main === module) {
  const input = readFileSync("./inputs/00.txt", "utf8");
  const solver = new Solver(input);
  console.log(solver.part1());
  console.log(solver.part2());
}

File: 2018/src/aoc2018/d00.clj
(ns aoc2018.d00
  (:require [clojure.string :as str]))

(defn parse [input]
  (str/split-lines (str/trim input)))

(defn part1 [input]
  (let [lines (parse input)]
    0))

(defn part2 [input]
  (let [lines (parse input)]
    0))

(defn -main []
  (let [input (slurp "inputs/00.txt")]
    (println (part1 input))
    (println (part2 input))))
